asapdiscovery-modeling/pyproject.toml

```diff
@@ -38,7 +38,11 @@ prep-mers-pdbs = "asapdiscovery.modeling.scripts.prep_mers_pdbs:main"
 prep-pdb-structures = "asapdiscovery.modeling.scripts.prep_pdb_structures:main"
 prep-proteins = "asapdiscovery.modeling.scripts.prep_proteins:main"
 split-oedu-and-save = "asapdiscovery.modeling.scripts.split_oedu_and_save:main"
+prepare-protein-csv = "asapdiscovery.modeling.scripts.prepare_protein_csv:main"
```

Comment:
Same here does this exist.

Reply:
done here: https://github.com/choderalab/asapdiscovery/pull/238/commits/f032d23b52cb58386ff9d408537ddf08889f47c5

Reply:
Thanks @apayne97!